apps/src/studio/studio.js

```diff
@@ -1633,6 +1633,8 @@ Studio.init = function(config) {
   Studio.tiles = [];
   Studio.tilesDrawn = false;
 
+  Studio.cloudStep = 0;
```

Comment:
We want clouds to continue from where they were last paused, so here makes sense.